app/src/main/java/com/bootcamp_android/parking_app/fragments/List.kt

```diff
@@ -50,7 +50,7 @@ class Lista : Fragment() {
         // TODO: Rename and change types and number of parameters
         @JvmStatic
         fun newInstance(param1: String,param2: String) =
-            Lista().apply {
+            List().apply {
```

Comment:
SGTM, nice using "Apply", I highly recommend to search about the other auxiliary functions in kotlin :)

Reply:
great